examples/mf-remote/rsbuild.config.ts

```diff
@@ -0,0 +1,26 @@
+import { pluginModuleFederation } from '@module-federation/rsbuild-plugin';
```

Comment:
So rslib use `rsbuild.config.ts` instead of `rslib.config.ts` ?

Reply:
> So rslib use `rsbuild.config.ts` instead of `rslib.config.ts` ?

this is app. 'react-component' is rslib project